src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -140,7 +143,7 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
         this.reset();
 
         // go back to the donor record page
-        if(isNotEmpty(this.donorRecordId)) {
```

Comment:
I think we need to add implementation for Cancel when the form gets loaded from the 'New Single Gift' button on the landing page. We can do that in a separate ticket.